code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -7,7 +7,7 @@
 	reagent_state = LIQUID
 	color = "#00bfff"
 	overdose = REAGENTS_OVERDOSE * 2
-	metabolism = REM * 0.5
```

Comment:
what's the purpose behind these granularly varied metabolic values? They were all the same before, that makes some math easier for medical players, is it causing problems?

Reply:
Speeding up heal recovery. It's rather slow right now.